tensorrt_llm/_torch/modules/fused_moe/moe_op_backend.py

```
@@ -0,0 +1,898 @@
+        gemm2_weights,
+        gemm2_weights_scale,
+        num_experts,
+        top_k,
+        n_group,
+        topk_group,
+        intermediate_size,
+        local_expert_offset,
+        local_num_experts,
+        routed_scaling_factor,
+        routing_method_type,
+        topk_weights=None,
```

Ruff (0.14.14)

[warning] Unused method argument: `topk_weights`

(ARG002)